### PR TITLE
Login

### PR DESCRIPTION
**Pull request by nhannn-2212**

### CODE REVIEW

**Comment by nhannn-2212** [comment]
image

**Comment by nhannn-2212** [comment]
ready

File: app/controllers/sessions_controller.rb [hunk 1]
```diff
@@ -0,0 +1,36 @@
+class SessionsController < ApplicationController
+  before_action :find_user_by_email, only: :create
+
+  def new
+    if logged_in?
+      redirect_to root_url
```
Comment (thaodtd-0665) [inline app/controllers/sessions_controller.rb:6]: thêm flash vì sao!

**Comment by nhannn-2212** [comment]
Description:
- Chức năng login, logout
Ticket: https://edu-redmine.sun-asterisk.vn/issues/24411
image
image

**Comment by nhannn-2212** [comment]
ready

**Pull request merged**